tools/convert_models/publish_model.py

```diff
@@ -21,18 +23,27 @@ def process_checkpoint(in_file, out_file):
         del checkpoint['optimizer']
     # if it is necessary to remove some sensitive data in checkpoint['meta'],
     # add the code here.
-    torch.save(checkpoint, out_file)
+    if torch.__version__ >= '1.6':
+        torch.save(checkpoint, out_file, _use_new_zipfile_serialization=False)
+    else:
+        torch.save(checkpoint, out_file)
+
     sha = subprocess.check_output(['sha256sum', out_file]).decode()
     if out_file.endswith('.pth'):
         out_file_name = out_file[:-4]
     else:
         out_file_name = out_file
-    final_file = out_file_name + f'-{sha[:8]}.pth'
+
+    current_date = datetime.datetime.now().strftime('%Y%m%d')
+    final_file = out_file_name + f'_{current_date}-{sha[:8]}.pth'
     subprocess.Popen(['mv', out_file, final_file])
 
 
 def main():
     args = parse_args()
+    out_dir = os.path.dirname(args.out_file)
+    if not os.path.exists(out_dir):
+        os.makedirs(out_dir)
```

Comment:
Be careful to create a series of dirs.
If users make a typo when inputting the out file path, it may produce unexpected results.

Reply:
Ok.  i raise a error when there is no tarhet dir, let the user make it manually